spec/Drupal/ParseComposer/InfoFileSpec.php

```diff
@@ -64,6 +64,7 @@ function it_understands_operators_in_constraints_in_d8()
         $fooInfo = '';
         $this->beConstructedWith('foo', $fooInfo, 8);
         $this->constraint('foo (>=1.5)')->shouldReturn(['drupal/foo' => '>=8.1.5']);
+        $this->constraint('system (>=8.1)')->shouldReturn(['drupal/system' => '>=8.1']);
```

Comment:
I think the version parser adds ``.0`` to the version constraint. ``shouldReturn(['drupal/system' => '>=8.1.0']);`` should work.